addons/mail/models/mail_template.py

```diff
@@ -65,10 +65,7 @@ def default_get(self, fields):
     auto_delete = fields.Boolean(
         'Auto Delete', default=True,
         help="This option permanently removes any track of email after it's been sent, including from the Technical menu in the Settings, in order to preserve storage space of your Odoo database.")
-    # contextual action
-    ref_ir_act_window = fields.Many2one('ir.actions.act_window', 'Sidebar action', readonly=True, copy=False,
```

Comment:
Why are you removing this ? This is quite handy in daily flow when using templates. Moreover this kind of changes should always be explained (with why you do the changes).

Reply:
After discussion with PDE, I will put the feature back but set it as accessible only for debug mode. The main reason for this feature removal was that its function is not very clear